demos/demo-big-maps/src/demo_big_maps/models.py

```diff
@@ -1,6 +1,4 @@
-from tortoise import fields
-from tortoise.fields.relational import ForeignKeyFieldInstance
-
+from dipdup import fields
```

Comment:
Maybe also replicate nested imports paths for consistency?
`from dipdup.fields.relational import ForeignKeyFieldInstance`

Reply:
It's not that easy. Let's discuss it later with other Tortoise stuff.